acrobat/styles/styles.css

```diff
@@ -206,6 +205,7 @@ div.how-to {
     "image" !important;
   }
   .how-to-heading {
+    padding-top: 44px;
```

Comment:
This is overridden by https://github.com/adobecom/dc/pull/294/files#diff-b343378297f83896f1a341f07ea0505bbcfa331963369b08eeef401b34d49747R396 .